examples/service-per-ns/indexedjob-controller.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This file can be deleted, right?

Reply:
Yes, that's right, as you can probably tell this started by copying indexed-job :smile: